controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -301,13 +300,20 @@ func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) (re
 		return reconcile.Result{}, err
 	}
 
-	err = r.setRookUninstallandCleanupPolicy(sc)
```

Comment:
Why are we removing this func call? This is required to tell rook about the kind of uninstall (force/graceful). We should not remove this.

Reply:
I pushed the wrong commit, this one was out of date

Reply:
Also note this moved to the newCephCluster() function. I couldn't figure out why we weren't always setting them to begin with.